Declare conf_file_set_enable_ipv6, so that it can be called without
an implicit declaration.  Improves C99 compatibility.

diff --git a/src/common/config-file.h b/src/common/config-file.h
index 23e1c89d488b789c..e710c202fe099f61 100644
--- a/src/common/config-file.h
+++ b/src/common/config-file.h
@@ -109,6 +109,7 @@ int conf_file_set_allow_inbound(const char *val, struct configuration *config);
 int conf_file_set_allow_outbound_localhost(const char *val, struct
 		configuration *config);
 int conf_file_set_isolate_pid(const char *val, struct configuration *config);
+int conf_file_set_enable_ipv6(const char *val, struct configuration *config);
 
 int conf_apply_socks_auth(struct configuration *config);
 
